packages/graph/src/graph2/graph.ts

```diff
@@ -691,8 +698,6 @@ export class Graph2 extends SVGZoomWidget {
                     .attr("class", "graphVertex")
                     .on("dblclick", function (this: SVGElement, d) {
                         d3Event().stopPropagation();
```

Comment:
I left this in to prevent zoom on double click.

Reply:
Would be better to move into the new double click code as:
```
d3Event().sourceEvent.stopPropagation();
```